src/pages/public_registration/action_accordions/DoiRequestAccordion.tsx

```diff
@@ -279,36 +289,40 @@ export const DoiRequestAccordion = ({
                 </ConfirmDialog>
               </>
             )}
-
-            <Modal
-              open={openRequestDoiModal}
-              onClose={toggleRequestDoiModal}
-              headingText={t('registration.public_page.request_doi')}
-              dataTestId={dataTestId.registrationLandingPage.tasksPanel.requestDoiModal}>
-              <Typography paragraph>{t('registration.public_page.request_doi_description')}</Typography>
-              <TextField
-                variant="outlined"
-                multiline
-                rows="4"
-                fullWidth
-                data-testid={dataTestId.registrationLandingPage.doiMessageField}
-                label={t('registration.public_page.message_to_curator')}
-                onChange={(event) => setMessageToCurator(event.target.value)}
-              />
-              <DialogActions>
-                <Button onClick={toggleRequestDoiModal}>{t('common.cancel')}</Button>
-                <LoadingButton
-                  variant="contained"
-                  data-testid={dataTestId.registrationLandingPage.tasksPanel.sendDoiButton}
-                  onClick={sendDoiRequest}
-                  loading={isLoadingData || isLoading !== LoadingState.None}>
-                  {t('common.send')}
-                </LoadingButton>
-              </DialogActions>
-            </Modal>
           </>
         )}
 
+        <Modal
```

Comment:
Skulle kanskje helst brukt `<ConfirmMessageDialog/>` som vi bruker flere andre plasser for dette. Men andre plasser er melding obligatorisk, mens det her er optinonal, så tar ikke bryet med å endre det der det trengs nå. Fortsetter derfor med dette sånn det var